lib/services/doctor-service.ts

```diff
@@ -83,5 +97,35 @@ class DoctorService implements IDoctorService {
 			this.$logger.out("TIP: To avoid setting up the necessary environment variables, you can use the Homebrew package manager to install the Android SDK and its dependencies." + EOL);
 		}
 	}
+
+	private verifyCocoaPods(): boolean {
+		this.$logger.out("Verifying CocoaPods. This may take more than a minute, please be patient.");
+
+		let temp = require("temp");
+		temp.track();
+		let projDir = temp.mkdirSync("nativescript-check-cocoapods");
+		this.$npm.install("tns-ios", projDir).wait();
```

Comment:
maybe we should try-catch this one as well. As we know - npm fails sometimes and in this case our doctor command will fail